src/sgame/sg_bot_util.cpp

```diff
@@ -1198,15 +1198,20 @@ void BotGetIdealAimLocation( gentity_t *self, botTarget_t target, vec3_t aimLoca
 {
 	//get the position of the target
 	BotGetTargetPos( target, aimLocation );
+	bool target_buildable = BotGetTargetType( target ) == entityType_t::ET_BUILDABLE;
+	//this retrieves the target's species, to aim at weak point:
+	// * for humans, it's the head (but code only applies an offset here, with the hope it's the head)
+	// * for aliens, there is no weak point, and human bots try to take missile's speed into consideration (for luci)
```

Comment:
```suggestion
	// * for aliens, there is no weak point, and human bots try to take missile's speed into consideration (for luci, pulse…)
```

Reply:
Well, this patch set only aim at changing nothing but make things more readable (and I learn a bit about desired code style).
Current code does not handle pulse rifle, blaster, flamer, barbs, pounces, ... at all, so changing the comment this way would result in what I hate the most: comments which lie.
When this code will be fixed so that it handle other weapons, then the comment shall be updated to reflect that, yes.